scss/sections/trade/_trade_respo.scss

```diff
@@ -9,34 +9,45 @@
             .container {
                 height: 100%;
                 .left_input {
-                    width:80%;
+                    position: absolute;
+                    width:100%;
                     top: 10%;
                     .amount_of_currency {
+                        width: 85%;
+                        left: 5%;
+                        top: 10%;
+                        position: relative;
+                    }
+                    .value {
                         left: 5%;
-                        position: absolute;
-                        .value {
-                            position: relative;
-                            left: 5%;
-                        }
                     }
                 }
                 .right_input {
```

Comment:
1.stop using reserved names -> input is reserved.
2. not miningful what it mean right input lol xD